charts/ia-case-api/values.preview.template.yaml

```diff
@@ -64,8 +64,7 @@ ia-case-payments-api:
   enabled: true #set to true if testing against a local branch
   java:
     imagePullPolicy: Always
-    image: hmctspublic.azurecr.io/ia/case-payments-api:latest
-    #    image: hmctspublic.azurecr.io/ia/case-payments-api:pr-343
+    image: hmctspublic.azurecr.io/ia/case-payments-api:pr-352 #:pr-259 or relevant PR number to change
```

Comment:
Are we going to merge this or are we going to point latest before merging to master?